NativeApp/src/components/Common/CustomDatePickerIOS/index.js

```diff
@@ -1,29 +1,122 @@
-import React from 'react';
-import { View, DatePickerIOS } from 'react-native';
+import React, { Component, Fragment } from 'react';
+import {
+  View,
+  Text,
+  DatePickerIOS,
+  Modal,
+  StyleSheet,
+  Button,
+  TouchableHighlight,
+} from 'react-native';
 import { PropTypes as PT } from 'prop-types';
+import moment from 'moment';
 
-const CustomDatePickerIOS = (props) => {
-  const { chosenDate, setDate, minimumDate } = props;
-  return (
-    <View>
-      <DatePickerIOS
-        date={chosenDate}
-        onDateChange={setDate}
-        mode="date"
-        minimumDate={minimumDate}
-      />
-    </View>
-  );
-};
-
-CustomDatePickerIOS.defaultProps = {
-  minimumDate: new Date(),
-};
-
-CustomDatePickerIOS.propTypes = {
-  chosenDate: PT.instanceOf(Date).isRequired,
-  setDate: PT.func.isRequired,
-  minimumDate: PT.instanceOf(Date),
-};
+class CustomDatePickerIOS extends Component {
+  static propTypes = {
+    chosenDate: PT.string.isRequired,
+    setDate: PT.func.isRequired,
+    minimumDate: PT.string,
+  };
+
+  static defaultProps = {
+    minimumDate: '',
```

Comment:
can we hardcode an actual minimum date here?

Reply:
We can, was previously set as `new Date()`. But the minimumDate prop was changed to receive a string instead. Changed it to be an empty string to the stop the prop type errors. Just forgot to add a value to it.

Was thinking of setting the default using `moment().format('YYYY-MM-DD')`. Therefore the current date is set as the default and the end picker can never be set to the day before. Unless the start date is changed.

Reply:
Same idea with the start date picker.